services/src/main/java/org/keycloak/services/managers/AuthenticationManager.java

```diff
@@ -103,6 +103,7 @@
     public static final String SET_REDIRECT_URI_AFTER_REQUIRED_ACTIONS= "SET_REDIRECT_URI_AFTER_REQUIRED_ACTIONS";
     public static final String END_AFTER_REQUIRED_ACTIONS = "END_AFTER_REQUIRED_ACTIONS";
     public static final String INVALIDATE_ACTION_TOKEN = "INVALIDATE_ACTION_TOKEN";
+    public static final String DEFER_CLEANUP = "DEFER_CLEANUP";
```

Comment:
The change of the deferred model of user session removal is unfortunately not justifiable in this case.